core/commands/storage/upload/upload/dc_repair_routing.go

```diff
@@ -0,0 +1,143 @@
+package upload
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/TRON-US/go-btfs/core/commands/cmdenv"
+	"github.com/TRON-US/go-btfs/core/commands/storage/helper"
+	uh "github.com/TRON-US/go-btfs/core/commands/storage/upload/helper"
+	"github.com/TRON-US/go-btfs/core/corehttp/remote"
+
+	"github.com/tron-us/go-btfs-common/crypto"
+	guardpb "github.com/tron-us/go-btfs-common/protos/guard"
+
+	cmds "github.com/TRON-US/go-btfs-cmds"
+)
+
+var StorageDcRepairRoutingCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Interact with Host repair requests and responses for negotiation of shards repair.",
+		ShortDescription: `
+Guard broadcasts repair request to potential nodes, after negotiation, nodes prepare the contracts for 
+such repair job, sign and send to the guard for confirmation `,
+	},
+	Subcommands: map[string]*cmds.Command{
+		"request":  hostRepairRequestCmd,
+		"response": hostRepairResponseCmd,
+	},
+}
+
+var hostRepairRequestCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Negotiate with hosts for repair jobs.",
+		ShortDescription: `
+This command sends request to mining host to negotiate the repair works.`,
+	},
+	Arguments: append([]cmds.Argument{
+		cmds.StringArg("peer-id", true, false, "Host Peer ID to send repair requests."),
```

Comment:
Actually I want to suggest this command can take many peerId as a []string to receive broadcast once, let us discuss which is better. 

https://github.com/TRON-US/go-btfs-guard/pull/360/files#diff-6bd9475267f3b0c5b7fbcdbc6c4adbbbR525